projects/scion/components.internal/package.json

```diff
@@ -38,5 +38,6 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/SchweizerischeBundesbahnen/scion-toolkit.git"
-  }
+  },
+  "sideEffects": false
```

Comment:
Do we have to configure this property? What is the default? I cannot imagine that the default is `true`.

Reply:
Set when generating a new Angular library